proteus-kits.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "bundler", "~> 1.7"
   spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rspec", "~> 2.6"
+  spec.add_development_dependency "rspec", "~> 3.3"
```

Comment:
There are differences between rspec 2.x and rspec 3.x.
They recommend upgrading to rspec 2.99 first.

I assume they have been addressed...